Resources/public/css/theme/views/testone.css

```diff
@@ -0,0 +1,23 @@
+.ez-testoneview-article + .ez-testoneview-article {
+    border-top: 1px solid #eee;
+}
+
+.ez-testoneview-article {
```

Comment:
To samo co w: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41733592